Tools/build/deepfreeze.py

```diff
@@ -491,7 +491,10 @@ def generate(args: list[str], output: TextIO) -> None:
 parser = argparse.ArgumentParser()
 parser.add_argument("-o", "--output", help="Defaults to deepfreeze.c", default="deepfreeze.c")
 parser.add_argument("-v", "--verbose", action="store_true", help="Print diagnostics")
-parser.add_argument('args', nargs="+", help="Input file and module name (required) in file:modname format")
+group = parser.add_mutually_exclusive_group(required=True)
+group.add_argument("-f", "--file", help="reads rule lines from a file")
```

Comment:
To be consistent with the other help messages:
```suggestion
group.add_argument("-f", "--file", help="read rule lines from a file")
```